src/pages/solutions/for-students.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: We help students 
+quickLinks:
+  - name: Data
+    anchor: data
+  - name: Basemaps and Imagery
+    anchor: imagery
+  - name: Conferences and Events
+    anchor: conferences
+  - name: Jobs and Internships
+    anchor: jobs
+  - name: Metadata
+    anchor: metadata
+  - name: Citations
+    anchor: citations
+
+---
+
+import Layout from '../../layouts/FullWidth.astro';
+import Section from '../../components/page/Section.astro';
+import QuickLinks from '../../components/page/QuickLinks.astro';
+
+<Layout title={frontmatter.title}>
+  <QuickLinks links={frontmatter.quickLinks}/>
+  <Section stripe title="Data">
+
+    `Data! Data! Data! I can't make bricks without clay!` - Sherlock Holmes
+
+    The <a href="/products/sgid/">State Geospatial Information Datasource</a> has a large <a href="/products/sgid-index/">catalog</a> of freely-available data you can use in your classwork, projects, or personal tinkering. 
+    
+    You can connect directly to the <a href="/products/open-sgid/">Open SGID</a> in your GIS software or access downloads and Esri web feature services through our <a href="https://opendata.gis.utah.gov">SGID Hub</a>.
+
+  </Section>
+  <Section title="Basemaps and Imagery">
+
+    Need high-res imagery for a research project? Students and faculty of Utah-funded schools can sign up for licensed access to <a href="/products/discover/">Discover</a>, our basemaps and imagery web services. 
```

Comment:
I wonder about switching this from them knowing they have a need to us telling them why they need it and how it would benefit what they are doing. Maybe we can mention fancy topics like computer vision etc